onnx/backend/test/case/model/sequence.py

```diff
@@ -13,6 +13,64 @@
 from typing import List, Optional, Text, Union
 
 
+def SequenceEmptyImpl():  # type: () -> List[Optional[np.ndarray]]
+    return []
+
+
+def SequenceConstructImpl(*tensors):  # type: (*np.ndarray) -> List[np.ndarray]
+    return list(tensors)
+
+
+def SequenceInsertImpl(sequence, tensor, position=None):
+    # type: (List[np.ndarray], np.ndarray, Optional[int]) -> List[np.ndarray]
+    if position is None:
+        position = len(sequence)
+    sequence.insert(position, tensor)
+    return sequence
+
+
+def SequenceAtImpl(sequence, position):
+    # type: (List[np.ndarray], int) -> np.ndarray
+    return sequence[position]
+
+
+def SequenceEraseImpl(sequence, position=None):
+    # type: (List[np.ndarray], Optional[int]) -> List[Optional[np.ndarray]]
+    if position is None:
+        position = -1
+    del sequence[position]
+    return sequence
+
+
+def SequenceLengthImpl(sequence):
+    # type: (List[np.ndarray]) -> np.int64
+    return np.int64(len(sequence))
+
+
+def SplitToSequenceImpl(tensor, split=None, axis=0, keepdims=1):
+    # type: (np.ndarray, Optional[Union[int, List[int]]], int, int) -> List[np.ndarray]
+    dim_size = tensor.shape[axis]
+    if split is None:
+        split = 1
+        split_indices = [i * split + 1 for i in range(dim_size) if i * split + 1 < dim_size]
+        if not keepdims:
+            results = np.array_split(tensor, split_indices, axis)
+            return [np.squeeze(res, axis) for res in results]
+    if np.isscalar(split):
+        split_indices = [i * split + 1 for i in range(dim_size) if i * split + 1 < dim_size]  # type: ignore
+    else:
+        split_indices = np.cumsum(split) + 1
+    return np.array_split(tensor, split_indices, axis)  # type: ignore
```

Comment:
Why do you ignore the type?

Reply:
some numpy methods don't work well with mypy